Used to enable both .info and .html documentation distribution, we also had to
drop cfcomdoc.css because that css file was not found in the original
distribution

--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -4,39 +4,10 @@
 # Info used in building the Info files
 info_TEXINFOS = cfengine-Reference.texinfo cfengine-Tutorial.texinfo
 infofiles = $(info_TEXINFOS:.texinfo=.info)
-MAKEINFOHTML = $(MAKEINFO) --html --no-split --css-include=cfcomdoc.css 
-
-# Info used in building and installing HTML files
-htmlfiles = $(info_TEXINFOS:.texinfo=.html)
-
-htmldir = $(pkgdatadir)/html
-html_DATA = $(htmlfiles)
-
-# Info used in building and installing Postscript files
-psfiles = $(info_TEXINFOS:.texinfo=.ps)
-pdffiles = $(info_TEXINFOS:.texinfo=.pdf)
-dvifiles = $(info_TEXINFOS:.texinfo=.dvi)
-
-docdir = $(pkgdatadir)/doc
-doc_DATA = $(psfiles) $(pdffiles)
 
 # Make sure these get distributed with everything else.
 
-EXTRA_DIST = cfagent.8 cfenvd.8 cfenvgraph.8 cfexecd.8 cfkey.8 cfrun.8 cfservd.8 cfengine.8 cfshow.8 cfetoolcheck.8   cfetooldump.8   cfetoolimport.8  cfetoolupdate.8 cfetoolcreate.8  cfetoolgraph.8  cfetoolinfo.8 cfetoolcheck.8   cfetooldump.8   cfetoolimport.8  cfetoolupdate.8 cfetoolcreate.8  cfetoolgraph.8  cfetoolinfo.8 texinfo.tex $(htmlfiles) $(infofiles) $(pdffiles)
-
-SUFFIXES = html latex pdf ps dvi
-
-%.html : %.texinfo
-	${MAKEINFO} -I $(srcdir) --html --no-split $<
-
-%.dvi : %.latex
-	${LATEX} $<
-
-%.dvi : %.texinfo
-	${TEXI2DVI} $<
-
-%.pdf : %.ps
-	${PS2PDF} $<
+EXTRA_DIST = cfagent.8 cfenvd.8 cfenvgraph.8 cfexecd.8 cfkey.8 cfrun.8 cfservd.8 cfengine.8 cfshow.8 cfetoolcheck.8   cfetooldump.8   cfetoolimport.8  cfetoolupdate.8 cfetoolcreate.8  cfetoolgraph.8  cfetoolinfo.8 cfetoolcheck.8   cfetooldump.8   cfetoolimport.8  cfetoolupdate.8 cfetoolcreate.8  cfetoolgraph.8  cfetoolinfo.8 texinfo.tex $(htmlfiles) $(infofiles)
 
 #
 # Some basic clean ups
